### PR TITLE
Return creator, not author, for pull request event

### PR DESCRIPTION
**Pull request by tkyi**
- Since creating events will require a `creator`, fixing bug in parseHook method to return creator instead of author for pull_request event
- Updated mock data to make sure new changes are working

Will need this information for creating events in webhooks in Screwdriver API

### CODE REVIEW

**Comment by FenrirUnbound** [comment]
In what scenarios is the author not the creator?

**Comment by tkyi** [comment]
@FenrirUnbound 
Scenario: if Min closes/reopens Dao's pull request

**Comment by FenrirUnbound** [comment]
Yup that scenario definitely fits the bill. 

The only scenarios I can't think of that this should cover are the `ping` and `badAction` payloads. When do those have a different sender?

File: test/data/github.pull_request.opened.json [hunk 1]
```diff
@@ -391,21 +391,21 @@
     "default_branch": "master"
   },
   "sender": {
-    "login": "baxterthehacker",
+    "login": "baxterthehacker2",
     "id": 6752317,
```
Comment (FenrirUnbound) [inline test/data/github.pull_request.opened.json]: Update

File: test/data/github.push.json [hunk 2]
```diff
@@ -142,21 +142,21 @@
     "email": "baxterthehacker@users.noreply.github.com"
   },
   "sender": {
-    "login": "baxterthehacker",
+    "login": "baxterthehacker2",
     "id": 6752317,
```
Comment (FenrirUnbound) [inline test/data/github.push.json]: I'm not going to call out each instance. Just make sure to update the `id` of the `sender` and the `avatar_url` across all the payloads

Comment (tkyi) [inline test/data/github.push.json]: fixed

File: test/data/github.pull_request.opened.json (hunk 1 above)
Comment (tkyi) [inline test/data/github.pull_request.opened.json]: fixed

**Pull request merged**